### PR TITLE
fix some unsatisfied.

### PR DESCRIPTION
**Pull request by chenhonggc**
fix some dissatisfied issues.

### CODE REVIEW

File: content/en/docs/tasks/run-application/scale-stateful-set.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ for which kubectl you're using.
 
 ### `kubectl scale`
 
-First, find the StatefulSet you want to scale. Remember, you need to first understand if you can scale it or not.
+First, find the StatefulSet you want to scale. Remember, you need to first understand if you can scale it or not. If you do not have a statefulSet, [create it first](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#creating-a-statefulset).
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/scale-stateful-set.md]: >do not have a statefulSet 

do not have a StatefulSet

File: content/en/docs/tasks/run-application/mysql-deployment.yaml [hunk 2]
```diff
@@ -9,11 +9,27 @@ spec:
     app: mysql
   clusterIP: None
 ---
+kind: PersistentVolume
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/mysql-deployment.yaml]: Add a new resource to the YAML is fine, but you have to change the tests/example_test.go file to change the test case as well. Travis won't feel happy about it unless the tests pass.

File: content/en/docs/tasks/run-application/run-single-instance-stateful-application.md [hunk 3]
```diff
@@ -49,14 +49,19 @@ Note: The password is defined in the config yaml, and this is insecure. See
 for a secure solution.
 
 {{< code file="mysql-deployment.yaml" >}}
+{{< code file="mysql-pv.yaml" >}}
+
+1. Deploy the pv and pvc of the YAML file:
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/run-single-instance-stateful-application.md]: nit: we avoid use 'pv' or 'pvc' in documentation text, pls consider use 'PV' and 'PVC' instead, or better yet, PersistentVolume and PersistentVolumeClaim respectively.

File: content/en/docs/tasks/run-application/run-single-instance-stateful-application.md [hunk 4]
```diff
@@ -163,8 +168,8 @@ specific to stateful apps:
 Delete the deployed objects by name:
 
 ```
-kubectl delete deployment,svc mysql
-kubectl delete pvc mysql-pv-claim
+kubectl delete -f https://k8s.io/docs/tasks/run-application/mysql-deployment.yaml
+kubectl delete -f https://k8s.io/docs/tasks/run-application/mysql-pv.yaml
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/run-single-instance-stateful-application.md]: For deletion, maybe the original commands are better. Fetching the yaml files only for deletion is an overkill.

**Comment by chenhonggc** [comment]
@steveperry-53 Please approve thanks.

**Comment by chenhonggc** [comment]
@tengqm I saw your name appeared in the approvers, Please approve the PR, Thanks.

File: content/en/docs/tasks/run-application/run-single-instance-stateful-application.md [hunk 5]
```diff
@@ -49,14 +49,19 @@ Note: The password is defined in the config yaml, and this is insecure. See
 for a secure solution.
 
 {{< code file="mysql-deployment.yaml" >}}
+{{< code file="mysql-pv.yaml" >}}
+
+1. Deploy the PV and PVC of the YAML file:
+
+        kubectl create -f https://k8s.io/docs/tasks/run-application/mysql-pv.yaml
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/run-single-instance-stateful-application.md]: can you verify if this command actually works? there were complaints that urls cannot be used as the '-f' parameter value.

Comment (chenhonggc) [reply content/en/docs/tasks/run-application/run-single-instance-stateful-application.md]: @tengqm `https://k8s.io/docs/tasks/run-application/mysql-pv.yaml` does not exist, I created it in the PR according to your suggestion, I use my github file `https://raw.githubusercontent.com/chenhonggc/kubernetes.github.io/format_style/content/en/docs/tasks/run-application/mysql-pv.yaml` to verify that this command is fine.

The cluster: `https://www.katacoda.com/courses/kubernetes/playground`
The output:
```
master $ kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3
465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:55:54Z", GoVersion:"go1.9.3", Compiler:
"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3
465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:
"gc", Platform:"linux/amd64"}

master $ kubectl create -f https://raw.githubusercontent.com/chenhonggc/kubernetes.github.io/format_style/c
ent/en/docs/tasks/run-application/mysql-pv.yaml
persistentvolume "mysql-pv-volume" created
persistentvolumeclaim "mysql-pv-claim" created

master $ kubectl get pv,pvc
NAME              CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                    STORAGECLASS   REASON    AGE
mysql-pv-volume   20Gi       RWO            Retain           Bound     default/mysql-pv-claim   manual              27s

NAME             STATUS    VOLUME            CAPACITY   ACCESS MODES   STORAGECLASS   AGE
mysql-pv-claim   Bound     mysql-pv-volume   20Gi       RWO            manual         27s 
```

Please review again. Thanks!

Comment (tengqm) [reply content/en/docs/tasks/run-application/run-single-instance-stateful-application.md]: Thanks for the confirmation.

**Pull request merged**